tests/suites/cloud/tests/preload/index.js

```diff
@@ -1,8 +1,35 @@
 "use strict";
+
+const request = require('request-promise');
+
 module.exports = {
   title: "Image preload test",
   run: async function (test) {
+
+    // if test is being done on physical DUT via the testbot, check that the preloaded application is working
+    if(this.workerContract.workerType !== `qemu`){
+      // we should be able to see the app starting.
+      const ip = await this.worker.ip(`${this.balena.uuid.slice(0, 7)}`);
+      
+      // create tunnel to DUT port 80
+      console.log(`Creating tunnel to DUT port 80...`)
+      await this.worker.createTunneltoDUT(`${this.balena.uuid.slice(0, 7)}`, 80, 8899);
+      await this.utils.waitUntil(async () => {
+        console.log(`Checking preloaded app is running... `)
+        let page = await request({
+          method: 'GET',
+          uri: `http://${ip}:80`,
+        })
+        return page.includes("Welcome to balena!") 
+      }, false);
+
+      test.ok(true, `Web page should be exposed on port 80 of DUT`)
+      // When we confirm the app has started, then re-enable internet access to DUT
+      await this.worker.executeCommandInWorker('sh -c "echo 1 > /proc/sys/net/ipv4/ip_forward"');
```

Comment:
Are we disabling internet for all worker types, but only re-enabling it for testbots?

Reply:
@klutchell this command will only ever disable the internet for testbots - that function doesn't do anything currently for the qemu worker because in the qemu scenario the core can't actually execute a command in the worker container ;/